src/platforms/dotnet/guides/azure-functions-worker/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Azure Functions Isolated Worker
+description: "Learn about Sentry's .NET integration with Azure Functions Isolated Worker."
+---
+
+Sentry provides an integration with Azure Functions Isolated Worker through the [Sentry.AzureFunctions.Worker NuGet package](https://www.nuget.org/packages/Sentry.AzureFunctions.Worker).
```

Comment:
```suggestion
Sentry provides an integration with Azure Functions through the [Sentry.AzureFunctions.Worker NuGet package](https://www.nuget.org/packages/Sentry.AzureFunctions.Worker).
```

<Note>

This package requires using the Azure Functions ["Isolated Worker Process"](https://learn.microsoft.com/azure/azure-functions/dotnet-isolated-process-guide) execution model.

</Note>

Reply:
https://www.martinfowler.com/bliki/TwoHardThings.html